schemas/20-cloud-extra.sql

```diff
@@ -53,17 +27,33 @@ PARTITION p2016_08 VALUES LESS THAN (TO_DAYS('2016-09-01')),
 PARTITION p2016_09 VALUES LESS THAN (TO_DAYS('2016-10-01')),
 PARTITION p2016_10 VALUES LESS THAN (TO_DAYS('2016-11-01')),
 PARTITION p2016_11 VALUES LESS THAN (TO_DAYS('2016-12-01')),
```

Comment:
Oops, no p2016_12 originally. Did this come from the grid version?

Reply:
Doesn't look like it, I must have made the Oops when adding new values